onnx2torch2/node_converters/pad.py

```diff
@@ -123,7 +123,14 @@ def _(node: OnnxNode, graph: OnnxGraph) -> OperationConverterResult:  # pylint:
     pads_name = node.input_values[1]
     pads = []
     if pads_name in graph.initializers or pads_name in graph._node_output_values:  # pylint: disable=W0212
-        pads = get_const_value(pads_name, graph).tolist()
+        # try and get padding arg value
+        try:
+            pads = get_const_value(pads_name, graph).tolist()
+        except KeyError as e:
+            raise (
```

Comment:
Raising a `ValueError` is better behaviour in my opinion, the user doesn't care about the KeyError, all that matters is that this Pad node has dynamic padding which we don't support.

Reply:
Should we still print out the caught error?

Reply:
changed to raise ValueError